test/hotspot/jtreg/runtime/Safepoint/TestAbortOnVMOperationTimeout.java

```diff
@@ -51,11 +51,9 @@ public static void main(String[] args) throws Exception {
             return;
         }
 
-        // These should definitely pass: more than a minute is enough for Serial to act.
+        // These should definitely pass: more than 3 minutes is enough for Serial to act.
         // The values are deliberately non-round to trip off periodic task granularity.
-        for (int delay : new int[]{63423, 12388131}) {
-            testWith(delay, true);
-        }
+        testWith(183423, true);
```

Comment:
"The values are ..." is no longer accurate since this has been changed to test a single value.

Reply:
Fixed.